### ISSUE TITLE
Launch Placeholder - WT / T search results

### ISSUE THREAD

**Issue by DHowett** [microsoft/terminal#12344]
Hold

**Comment by Stanzilla** [comment]
1.13.10336.0 is still called Windows Terminal for me and therefore also can be found in the start menu, quirk on my side or did the rename not actually ship in 1.13.10336.0?

**Comment by DHowett** [comment]
Explorer caches the names >_<

**Comment by Stanzilla** [comment]
> Explorer caches the names >_<

Oh, ok. About window says "Windows Terminal" as well though

**Comment by The-MAZZTer** [comment]
I updated from 1.12 preview to 1.13 preview and my start menu shortcut is still "Windows Terminal Preview".

Does this change only apply to new installs?

**Comment by The-MAZZTer** [comment]
> Oh, ok. About window says "Windows Terminal" as well though

I checked the source code and it looks like that is pulled from the package display name. Microsoft Store still lists it with "Windows" on there so it depends when they update it.

Or they may want to decouple those if there's a specific reason to keep "Windows" on the Store name.

**Comment by vadimkantorov** [comment]
Also, if one starts to type in search bar before Indexing service starts up, one cannot even find it by exact name used in the start menu. I wish Start Menu enabled search of its items as a separate indexing procedure and not having to wait some indefinite time for Windows Indexing service to start up (on some slow systems it can take even minutes)

**Comment by akirataguchi115** [comment]
Same thing here. Can be opened from the Microsoft Store but not found from the start menu.

**Comment by bearoxo** [comment]
Yup.. Same. Installed via winget, no shortcuts created, via choco, no shortcuts created, via Microsoft Store, still no shortcuts....

**Comment by akirataguchi115** [comment]
I actually managed to solve this. It is called "wt" in all languages but for example in Finnish it is found as "Pääte", which is the translation of the application name. Could you try this, @DHowett ?

**Comment by vadimkantorov** [comment]
I think it would be best to have some aliases available in windows using any language pack: both "wt" and "terminal". Ideally, they should be searchable before Windows Indexing service starts up...

**Comment by jredfox** [comment]
I have previously stated the same thing. “wt” , “windowsterminal” “terminal” with or without lowercase variants

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by akirataguchi115** [comment]
I don't mind. I think there should only be the local version alias searchable. This would be good for other languages' developments than en-us.
